javaparser-core/src/main/java/com/github/javaparser/ast/expr/StringLiteralExpr.java

```diff
@@ -78,6 +78,10 @@ public boolean remove(Node node) {
         return super.remove(node);
     }
 
+    public String asString() {
+        return value;
+    }
```

Comment:
That's not right, is it? This should have the same (un)escpaing as `asChar()`.

Interesting question: so what should `StringLiteralExpr(String)` do?